controllers/release/release_adapter.go

```diff
@@ -228,24 +228,41 @@ func (a *Adapter) EnsureSnapshotEnvironmentBindingIsCreated() (results.Operation
 // EnsureTargetContextIsSet is an operation that will ensure that the targetContext of the adapter is not nil. It will
 // set it to a context from the workspace in the ReleasePlan if one is defined or to its own context if not.
 func (a *Adapter) EnsureTargetContextIsSet() (results.OperationResult, error) {
-	if a.targetContext == nil {
+	if a.targetContext != nil {
```

Comment:
@johnbieren Are you ok with removing this check? This is not a real condition as the adapter is created per reconcile.